tools/tool-base/src/test/java/io/spine/tools/protoc/MethodsTest.java

```diff
@@ -62,11 +60,11 @@ void addMultipleFilePatterns() {
         String pattern = "testPattern";
         String interfaceName = "io.spine.test.TestInterface";
 
-        GeneratedMethods defaults = new GeneratedMethods();
+        Methods defaults = new Methods();
         MessageSelectorFactory messages = defaults.messages();
-        defaults.applyFactory(interfaceName, messages.inFiles(suffix(pattern)));
-        defaults.applyFactory(interfaceName, messages.inFiles(prefix(pattern)));
-        defaults.applyFactory(interfaceName, messages.inFiles(regex(pattern)));
+        defaults.applyFactory(interfaceName, messages.inFiles(MessageSelectorFactory.suffix(pattern)));
```

Comment:
It's the same question on static imports of the `MessageSelectorFactory`methods.